vimeo/iris
# Dsys 5 slider number input

## What this PR does 
Adds UI for arrows up/down for the slider editable input

## Screenshots & Recordings 

https://github.com/vimeo/iris/assets/9106375/6eb41187-762a-4c0b-a930-d1036921b2f5


## How it does that 
* Separate the EditableInput into a component
* Added Arrows component
* Handle click up/down logic

## Testing 
* Hover editable input in the slider
* See arrows up and down
* Click on them and see value is changing
